controllers/persistentvolume/persistentvolume_controller_test.go

```diff
@@ -230,7 +231,7 @@ var _ = Describe("PersistentVolume Controller", func() {
 				if pvName == PersistentVolumeForTopology {
 					pv.Spec.ClaimRef.Name = PersistentVolumeClaimForTopology
 				}
-				if pvName == secondPersistentVolume {
+				if pvName == secondPersistentVolume || pvName == secondPersistentVolumeForTopology {
```

Comment:
consider using meaningful names for the PVs, it will help us understand the scenario better.